### DISCUSSION TITLE
Approach circles blend into the background

### DISCUSSION THREAD

**Discussion by soopax** [ppy/osu#19083] [Ideas] [answered]
It's sometimes hard to see when the circles are supposed to be hit.
This is especially apparent if background dim is set to 100% and there are black hit circles.
Maybe something could be done about this?
See this screenshot for an example:
![osu_2022-07-07_00-23-34](https://user-images.githubusercontent.com/101510334/178487511-ff8e5c3e-fe55-406e-aadc-3d4209bded48.png)

**Comment by peppy** [comment, marked as answer]
Have you considered changing background dim or disabling beatmap skins?

Beatmaps set the combo colours usually to match the background. You are disabling part of this and thus breaking the beatmap. I don't think this is something we should be fixing from our end, but you can definitely do something about it by changing one or more settings.
